biothings/web/query/builder.py

```diff
@@ -29,18 +29,21 @@ class implementations or not defined.
         facet_size: int, maximum number of agg results.
 
 """
-import logging
-import os
-import re
+
 from collections import UserString, namedtuple
 from copy import deepcopy
 from random import randrange
+import logging
+import os
+import re
+from typing import Iterable
 
-import orjson
 from elasticsearch_dsl import MultiSearch, Q, Search
 from elasticsearch_dsl.exceptions import IllegalOperation
+import orjson
 
 from biothings.utils.common import dotdict
+from biothings.web.settings.default import ANNOTATION_DEFAULT_REGEX_PATTERN
```

Comment:
importing directly from `default` can be a problem, as it does not allow users to override its value in `config.py` or `config_web.py`. You might want to check `namespace.py` to see how config parameters are passed, e.g. `ANNOTATION_ID_REGEX_LIST`.

Reply:
Gotcha good to know